### DISCUSSION TITLE
Profile Option: More Pin

### DISCUSSION THREAD

**Discussion by criticalsoft** [community/community#7690] [Other Feature Feedback, Questions, & Ideas]
Six is not enough for some people.
(They can't pin some important project more than 6)

**Comment by radar-blip** [comment]
Its just one of those things. 
Or 6. 
Anyway its better than 5.

**Comment by Tostisto** [comment]
I think 6 is enough. But if this is not enough for you, you can use the profile readme and there you can pin as much as you want. For example, you can use this: [extra pins ](https://github.com/anuraghazra/github-readme-stats#github-extra-pins)

**Comment by criticalsoft** [comment]
Yeap, 6 is enough, Since found i spent too much time on my project.
How to close this discussion Or close it ?

**Comment by ChrisCarini** [comment]
@criticalsoft - I think you can opt to select an answer.

Sounds like the best solution you can get working right away would be:

1. Use [anuraghazra/github-readme-stats#github-extra-pins](https://github.com/anuraghazra/github-readme-stats#github-extra-pins)
2. Place two side-by-side in a table to mimic the current GitHub UI (see suggestion https://github.com/anuraghazra/github-readme-stats/issues/1490 from the creator)

**Comment by thuongtruong1009** [comment]
You can try with ***[extra-pins](https://github.com/anuraghazra/github-readme-stats)***
Add the syntax in your ***README.md*** file this line: 
```bash
![](https://github-readme-stats.vercel.app/api/pin/?username=your_name&repo=your_repo) 
```
And it will be displayed like below:
[![](https://github-readme-stats.vercel.app/api/pin/?username=thuongtruong1009&repo=bee-music)](https://github.com/thuongtruong1009/bee-music)
